recipes/salmon/build_config.yaml

```diff
@@ -0,0 +1,6 @@
+c_compiler:
+  - gcc      # [linux]
+  - clang    # [osx]
```

Comment:
Rob can you try to replace clang here with GCC as well?

Reply:
Ok; just pushed.  I didn't even know that building with GCC on OSX was an option here :).